Doppler.Currency/appsettings.json

```diff
@@ -43,9 +43,10 @@
       "CurrencyCode": "MXN",
       "CurrencyName": "Peso Mexicano"
     }
-  },
-  "SlackHook": {
-    "Url": "[SECRET KEY]",
-    "Text": "Doppler Currency Service can't get the currency today, Please check Html format"
   }
+  /* Add Property SlackHook if you want Slack integration
+    "SlackHook": {
+      "Url": "",
+      "Text": ""
+    }*/
```

Comment:
I feel that comment in the `appsettings.json` is not a good approach.
What do you think about leve el `Url` with a empty value as a not configured service?

Reply:
I think that it is _more correct_  avoiding the property _SlackHook_ when we do not want to use it. It is more explicit than assuming that behavior when the URL is empty.

But I am not entirely happy, this comment is a smell that our decision is not entirely well. 

@elsupergomez what do you think a different approach, something like this:

```json
    "SlackHook": {
      "Enabled": false,
      "Url": "",
      "Text": ""
    }
```